[2024-12-19T23:56:14.982+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.data_validation_structured manual__2024-12-19T23:55:50.668627+00:00 [queued]>
[2024-12-19T23:56:14.992+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.data_validation_structured manual__2024-12-19T23:55:50.668627+00:00 [queued]>
[2024-12-19T23:56:14.993+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-19T23:56:15.008+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): data_validation_structured> on 2024-12-19 23:55:50.668627+00:00
[2024-12-19T23:56:15.015+0000] {standard_task_runner.py:60} INFO - Started process 24553 to run task
[2024-12-19T23:56:15.017+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'data_validation_structured', 'manual__2024-12-19T23:55:50.668627+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpa12j0k40']
[2024-12-19T23:56:15.020+0000] {standard_task_runner.py:88} INFO - Job 214: Subtask data_validation_structured
[2024-12-19T23:56:15.071+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.data_validation_structured manual__2024-12-19T23:55:50.668627+00:00 [running]> on host 1d3bd1c7da18
[2024-12-19T23:56:15.159+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='data_validation_structured' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T23:55:50.668627+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-19T23:55:50.668627+00:00'
[2024-12-19T23:56:16.826+0000] {logging_mixin.py:188} INFO - structured_validation executed successfully.
[2024-12-19T23:56:16.827+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-19T23:56:16.837+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=data_validation_structured, execution_date=20241219T235550, start_date=20241219T235614, end_date=20241219T235616
[2024-12-19T23:56:16.878+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-19T23:56:16.905+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
